code/src/components/AddTodo.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import { todoList } from "reducers/todoList"
+
+export const AddTodo = () => {
+  const [todo, setTodo] = useState("")
+  const dispatch = useDispatch()
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    dispatch(todoList.actions.addTodo(todo))
+    setTodo("")
+  }
```

Comment:
to make this even better you could check so that the todo variable isn't empty before saving it as a todo? right now you could create empty todos.
```
const handleSubmit = (event) => {
  event.preventDefault()
  
  if (todo) {
    dispatch(todo.actions.addItem(todo))
    setTodo('') 
  }
}
```